gen_config/model.conf

```diff
@@ -1 +1 @@
-java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file:///<KILLBILL_REPO>/jaxrs/src/main/java/com/ -o /tmp/output -t PHP_CLIENT_API -p com.ning.billing.jaxrs.json
+java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file://<KILLBILL_REPO>/jaxrs/src/main/java/org -o /tmp/output -t PHP_CLIENT_API -p org.killbill.billing.jaxrs.json
```

Comment:
Yup, but it looks like you can put either `killbill/jaxrs/src/main/java/` or `killbill/jaxrs/src/main/java/org`, the code is smart enough to find the files regardless.

Same for `file://` vs `file:///`, code supports both.

Anyways, not really important, it was just more a consistency thing.